source/Hardpoint.cpp

```diff
@@ -227,7 +228,7 @@ void Hardpoint::Fire(Ship &ship, vector<Projectile> &projectiles, vector<Visual>
 	projectiles.emplace_back(ship, start, aim, outfit);
 
 	// Create any effects this weapon creates when it is fired.
-	CreateEffects(outfit->FireEffects(), start, ship.Velocity(), aim, visuals);
+	CreateEffects(outfit->FireEffects(), visualStart, ship.Velocity(), aim, visuals);
```

Comment:
If we create projectile effects at a different position than they actually are wouldn't that mean that we might see a visual bug where a very fast projectile hits something while its effect lags behind and appears as if it hasn't hit the ship?

Reply:
This concerns me as well, but Ferocious pointed out that maybe normal projectiles need the 0.5x velocity.  I'll have to look at what normal projectiles do without it in addition to what the Dragonflame Cannon does.